src/components/charts/sankey/sankey-tooltip/sankey-tooltip-component.jsx

```diff
@@ -20,10 +20,10 @@ class SankeyTooltip extends PureComponent {
                 <div key={node.name}>
                   <div className={styles.tooltipHeader}>
                     <span className={styles.targetName}>
-                      {
-                        node.payload.payload &&
-                          node.payload.payload.target &&
-                          node.payload.payload.target.name
+                      {node.payload.payload && 
+                        node.payload.payload.target &&
+                          `${ node.payload.payload.target.name } 
+                           ${ node.payload.payload.timeframes ? node.payload.payload.timeframes : "" }`
```

Comment:
Just a matter of taste but maybe instead of a ternary a logical AND could be used:
`${node.payload.payload.timeframes && node.payload.payload.timeframes}`

Reply:
Yes, I thought about that but when I use logical operator `&&` in string template, it will return `undefined`, if `node.payload.payload.timeframes` doesn't exist. So I had to return an empty string in this case, because in other case it will set `undefined` as a text of the tooltip. I don't know if there is a nicer syntax for that, any thoughts?

Reply:
My bad :>D
Feel free to merge it.

Reply:
Ok, thanks :)